### DISCUSSION TITLE
Error: The repository 'https://aquasecurity.github.io/trivy-repo/deb trixie Release' does not have a Release file.

### DISCUSSION THREAD

**Discussion by j-lakeman** [aquasecurity/trivy#6518] [Q&A] [answered]
### Description

On Debian Sid I can't install trivy from the repo in the documentation.

### Desired Behavior

Install the package without any flaws.

### Actual Behavior

Upon package update:

`Ignored:   https://aquasecurity.github.io/trivy-repo/deb trixie InRelease`
`Error: https://aquasecurity.github.io/trivy-repo/deb trixie Release
  404  Not Found [IP: 2606:50c0:8002::153 443]`
`Error: The repository 'https://aquasecurity.github.io/trivy-repo/deb trixie Release' does not have a Release file.`

### Reproduction Steps

```bash
1. try to set up the Debian repo as described there https://aquasecurity.github.io/trivy/v0.50/getting-started/installation/
2. `sudo apt update`
3. see above “Actual Behavior”
```

### Target

None

### Scanner

None

### Output Format

None

### Mode

None

### Debug Output

```bash
n/a
```

### Operating System

Debian Sid

### Version

```bash
n/a
```

### Checklist

- [ ] Run `trivy image --reset`
- [ ] Read [the troubleshooting](https://aquasecurity.github.io/trivy/latest/docs/references/troubleshooting/)

**Comment by DmitriyLewen** [comment, marked as answer]
Hello @j-lakeman 
Thanks for your report!

We only have deb packages for stable releases.
Debian `trixie` is currently being tested - https://www.debian.org/releases/index.en.html.
After release we will try to add a new deb package as soon as possible.

Regards, Dmitriy

**Reply by j-lakeman** [reply]
Hey @DmitriyLewen, thanks for your swift reply!

That's a pity as many people, especially developers and contributors are using _Trixie_ or _Sid_ IMHO. What do you mean by release? Debian release or trivy release?

PS: I manually downloaded the provided `.deb` and its working as expected for me under _Sid_.

**Reply by DmitriyLewen** [reply]
Hello @j-lakeman. 
I meant Trixie Debian release.

> That's a pity as many people, especially developers and contributors are using Trixie or Sid IMHO

we have not received any issues from users asking us to add `sid` releases.
You can use [GH Binaries](https://aquasecurity.github.io/trivy/v0.50/getting-started/installation/#install-from-github-release-official) (we have script for easy installation/update)
